### ISSUE TITLE
npm

### ISSUE THREAD

**Issue by thadclay** [npm-dom/domquery#6]
Any guidance on how to proceed using this package with node now that its not in npm anymore?

**Comment by vvo** [comment]
```js
{
  "dependencies": {
    "domquery": "npm-dom/domquery"
  } 
}
```

Should do it

**Comment by thadclay** [comment]
@vvo That doesn't work without updating all the dependent modules that domquery uses. I can submit a pull request to this project and all its dependents if the maintainer(s) are open to it.

**Comment by vvo** [comment]
I really dont know I think it's not really heavily maintained

**Comment by TehShrike** [comment]
I've fixed all the dependencies and re-published to npm.  The current version has been published as 3.0.0.
